test/sqllogictest/table_func.slt

```diff
@@ -38,6 +38,13 @@ SELECT generate_series FROM generate_series(2, 4)
 3
 4
 
+query I
+SELECT generate_series FROM generate_series(2, 4) WITH ORDINALITY
```

Comment:
Also needs an `ORDER BY` to be deterministic.

Reply:
Thanks. Fixed up all of these tests which were missing `rowsort`s.